PDF Sharp 'FontFamily' is defined in an assembly that is not referenced

## Description
I'm getting an error CS0012: The type 'FontFamily' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Drawing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.

I can make this go away in visual studio with the "add reference" action but this only solves the issue in visual studio, it still shows up in Unity's console
![image](https://user-images.githubusercontent.com/3460556/105089284-c3dfb180-5a51-11eb-8bda-98ba6c4757cb.png)

before:
![image](https://user-images.githubusercontent.com/3460556/105088485-cdb4e500-5a50-11eb-8fb1-daa61635fad2.png)

after:
![image](https://user-images.githubusercontent.com/3460556/105088611-f937cf80-5a50-11eb-833d-80e91f2cd88d.png)



- **NuGet Package:** PdfSharp Version 1.50.5147
- **NuGetForUnity Version:** 2.0.1
- **Unity Version:** 2020.2.1
- **Operating System:** Windows 10

--- a/README.md
+++ b/README.md
@@ -235,4 +235,4 @@ To import one of the System libraries and resolve the error you need to create a
 -r:System.Net.Http.dll
 ```
 
-and place it inside the containig the project that requires the library e.g. the `Assets` folder. It can also be placed in a folder that contains a `.asmdef` so the reference is only added to the sub-project. NuGetForUnity also uses a `csc.rsp` file see [csc.rsp](src/NuGetForUnity/Editor/csc.rsp).
+and place it inside the containing project that requires the library e.g. the `Assets` folder. It can also be placed in a folder that contains a `.asmdef` to only add the reference to the sub-project. For example NuGetForUnity also uses a `csc.rsp` file see [csc.rsp](src/NuGetForUnity/Editor/csc.rsp).

--- a/README.md
+++ b/README.md
@@ -220,3 +220,19 @@ As mentioned in the message this error can be suppressed by changing a setting. 
 <summary>Screenshot of Player Settings</summary>
 <img als="Assembly Version Validation settings page" src="docs/screenshots/assembly-version-validation-setting.png" height="550px" />
 </details>
+
+## Missing System libraries
+
+When targeting .Net Framework, the 'Api Compatibility Level' setting under _Edit_ -> _Project Settings_ -> _Player_ -> _Other Settings_ is set to .NET Framework, Unity doesn't include some System libraries by default. See also [Unity documentation about system libraries](https://docs.unity3d.com/Manual/dotnetProfileAssemblies.html) and [Unity documetnation about .Net Profiles](https://docs.unity3d.com/Manual/dotnetProfileSupport.html). Libraries that are not imported by default are e.g. `System.Drawing`, `System.IO.Compression`, `System.IO.Compression.FileSystem` and `System.Net.Http`. If you try to use one of this libraries you get a error like:
+
+```
+The type `HttpClient` is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
+```
+
+To import one of the System libraries and resolve the error you need to create a `csc.rsp` file with the content e.g. to import `System.Net.Http`
+
+```
+-r:System.Net.Http.dll
+```
+
+and place it inside the containig the project that requires the library e.g. the `Assets` folder. It can also be placed in a folder that contains a `.asmdef` so the reference is only added to the sub-project. NuGetForUnity also uses a `csc.rsp` file see [csc.rsp](src/NuGetForUnity/Editor/csc.rsp).

